data_structures/binary_tree/lazy_segment_tree.py

```diff
@@ -72,20 +72,6 @@ def query(self, idx, l, r, a, b): #query(1, 1, N, a, b) for query max of [a,b]
 
     def showData(self):
         showList = []
-        for i in range(1,N+1):
+        for i in range(1,(self.N)+1):
```

Comment:
Unecessary parentheses
```suggestion
        for i in range(1,self.N+1):
```